staging/src/k8s.io/pod-security-admission/test/fixtures_sysctls.go

```diff
@@ -41,14 +41,15 @@ func init() {
 				tweak(p, func(p *corev1.Pod) { p.Spec.SecurityContext.Sysctls = nil }),
 				// sysctls with name="kernel.shm_rmid_forced" ,"net.ipv4.ip_local_port_range"
 				// "net.ipv4.tcp_syncookies", "net.ipv4.ping_group_range",
-				// "net.ipv4.ip_unprivileged_port_start"
+				// "net.ipv4.ip_unprivileged_port_start", "net.ipv4.ip_local_reserved_ports"
 				tweak(p, func(p *corev1.Pod) {
 					p.Spec.SecurityContext.Sysctls = []corev1.Sysctl{
 						{Name: "kernel.shm_rmid_forced", Value: "0"},
 						{Name: "net.ipv4.ip_local_port_range", Value: "1024 65535"},
 						{Name: "net.ipv4.tcp_syncookies", Value: "0"},
 						{Name: "net.ipv4.ping_group_range", Value: "1 0"},
 						{Name: "net.ipv4.ip_unprivileged_port_start", Value: "1024"},
+						{Name: "net.ipv4.ip_local_reserved_ports", Value: "1024 4999"},
```

Comment:
Why use ip_local_reserved_ports "1024 4999" here?
- centos: default empty
- redhat: default "1024 4999"

Reply:
it's empty on my Ubuntu 18.04

Reply:
4999 looks like a “magic number to me.

BTW it's a comma separated list. So if we do this we might want to allow setting a more complex value (eg, an array at API level).

Reply:
> 4999 looks like a “magic number to me.

Me too. This is a magic number from Redhat. And centos/ubuntu leave it empty. 
I use Redhat's default value in this test case only, not as a default value.

> BTW it's a comma separated list. So if we do this we might want to allow setting a more complex value (eg, an array at API level).

It is a string at the API level. It is comma-separated on the kernel side.